Week2/homework/maartjes-work.js

```diff
@@ -44,18 +44,29 @@ const tuesday = [
 
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
+const timeThreshold = 120;
+
+// function computeEarnings(tasks, hourlyRate) {
+//   return tasks
```

Comment:
I see what you are trying to do here and it works fine. However, you missed one small point from homework which is `Map the tasks to durations in hours.`. It's equally important that you follow all instructions provided in the homework. 😉

Reply:
I missed that point and shifted that step in `reduce` part. Now added `map` step for that requirement. Thanks for drawing attention to that point